src/sql/workbench/services/tableDesigner/browser/tableDesignerComponentInput.ts

```diff
@@ -185,7 +190,7 @@ export class TableDesignerComponentInput implements DesignerComponentInput {
 			return;
 		}
 		const dialog = this._instantiationService.createInstance(TableDesignerPublishDialog);
-		const result = await dialog.open(report);
+		const result = await dialog.open(<any>report.report);
```

Comment:
@abist FYI. I have do this as a temporary solution to make the report work, If I merge this PR one before yours, you will need to resolve a conflict.